Sources/Public/Animation/LottieAnimationLayer.swift

```diff
@@ -0,0 +1,1210 @@
+//
+//  LottieAnimationLayer.swift
+//  Lottie
+//
+
+import Foundation
+import QuartzCore
+
+// MARK: - LottieAnimationLayer
+
+/// A CALayer subclass for rendering Lottie animations.
+/// All functionality is also available in a UIView as `LottieAnimationView`.
+
```

Comment:
There shouldn't be a blank line between doc comments and their declaration -- I think this prevents the doc comment from being properly associated with the declaration

```suggestion
```